%YAML:1.0
---
camera_matrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 4.8032099071196194e+02, 0., 3.9875248806041247e+02, 0.,
       4.8002081091904893e+02, 4.0053640375536133e+02, 0., 0., 1. ]
distortion_coefficients: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ -1.9526472315724411e-02, 1.0629252312572395e-01,
       -6.7665129406160721e-04, -8.1985437562373520e-04,
       -1.4494254253660807e-01 ]
